mail-client/claws-mail/claws-mail-3.11.1-r1.ebuild

```diff
@@ -99,6 +99,10 @@ RDEPEND="${COMMONDEPEND}
 	app-misc/mime-types
 	x11-misc/shared-mime-info"
 
+src_prepare() {
+	epatch "${FILESDIR}/${P}_RSSyl-encodings-fix.patch"
```

Comment:
You can either push a second commit to the branch, or amend the existing commit and force-push.